app/src/main/java/com/google/android/apps/exposurenotification/nearby/ExposureNotificationClientWrapper.java

```diff
@@ -0,0 +1,145 @@
+package com.google.android.apps.exposurenotification.nearby;/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import android.content.Context;
+import com.google.android.gms.nearby.Nearby;
+import com.google.android.gms.nearby.exposurenotification.ExposureConfiguration;
+import com.google.android.gms.nearby.exposurenotification.ExposureInformation;
+import com.google.android.gms.nearby.exposurenotification.ExposureNotificationClient;
+import com.google.android.gms.nearby.exposurenotification.ExposureSummary;
+import com.google.android.gms.nearby.exposurenotification.TemporaryExposureKey;
+import com.google.android.gms.tasks.Task;
+import com.google.android.gms.tasks.Tasks;
+import com.google.common.collect.Lists;
+import java.io.File;
+import java.util.List;
+
+/**
+ * Wrapper around {@link com.google.android.gms.nearby.Nearby} APIs.
+ */
+public class ExposureNotificationClientWrapper {
```

Comment:
Needs to be refactored to kotlin and classic repository pattern. This is not acceptable.

Reply:
Can be done in next task branch